streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala

```diff
@@ -26,7 +26,7 @@ private[ui] abstract class BatchTableBase(tableId: String) {
   protected def columns: Seq[Node] = {
     <th>Batch Time</th>
       <th>Input Size</th>
-      <th>Scheduling Delay</th>
+      <th>Streaming Scheduling Delay</th>
```

Comment:
As discussed offline, the extra wide column look weird. Rather remove `Streaming` and in a future PR add tooltips to explain each column title.